app/containers/SearchPage/SearchView.tsx

```diff
@@ -182,7 +182,7 @@ const COLUMN_NAMES = fromPairs(
 
 const changePage = (pageNumber: number) => (params: SearchParams) => ({
   ...params,
-  page: pageNumber,
+  page: Math.min(pageNumber, Math.ceil(MAX_WINDOW_SIZE / params.pageSize) - 1),
```

Comment:
To limit the pagecounter on the CrumbsBar to be only 10,000, aka the max window size allowed for Searchkick. We went over before why we shouldn't allow more than 10k results.